src/server/api/lib/utils/syncES.js

```diff
@@ -0,0 +1,77 @@
+require('dotenv').config();
+
+const { client, usersIndex } = require('../../../config/elastic');
+const knex = require('../../../config/db');
+
+const findIndex = async (index) => {
+  const indexExists = await client.indices.exists({
+    index: index,
+  });
+  return indexExists.body;
+};
+
+const createIndex = async (index) => {
+  if (await findIndex(index)) {
+    await client.indices.delete({
+      index: index,
+    });
+  }
+
+  await client.indices.create({
+    index: index,
+    body: {
+      mappings: {
+        dynamic: 'strict',
+        properties: {
+          fullName: {
+            type: 'search_as_you_type',
+            fields: {
+              keyword: {
+                // looks like this subfield is not being placed into mapping
+                type: 'keyword',
+                ignore_above: 256,
+              },
+            },
+          },
+          email: {
+            type: 'keyword',
+          },
+        },
+      },
+    },
+  });
+  const indexMapping = await client.indices.get({
+    index: index,
+  });
+  console.log(indexMapping);
+  console.log(`'${index}' index created! `);
+};
+
+const moveUSersfromDBtoES = async () => {
+  const DBusers = await knex('users').select(
+    'users.id',
+    'users.fullName',
+    'users.email',
+    'users.createdOn',
+    'users.firebaseUId',
+  );
+  await DBusers.map(async (user) => {
+    const newESuser = {
+      fullName: user.fullName, // fullname
+      email: user.email,
+    };
+    //await?
+    await client.index({
+      id: user.id,
+      index: usersIndex,
+      body: newESuser,
+    });
+  });
+};
+
+//createIndex(usersIndex);
+
+moveUSersfromDBtoES().then(() => {
+  console.log('moving from DB to ES done!');
+  knex.destroy();
+});
```

Comment:
I think something like this might work:

```suggestion
createIndex(usersIndex).then(() =>
moveUSersfromDBtoES().then(() => {
  console.log('moving from DB to ES done!');
  knex.destroy();
});
});
```

(The indentation in my suggestion is a bit off, so make sure to fix that and also try it out locally to see if it actually works 😄 )